go/service/wallet.go

```diff
@@ -27,7 +26,7 @@ func newWalletHandler(xp rpc.Transporter, g *libkb.GlobalContext) *walletHandler
 		BaseHandler:  NewBaseHandler(g, xp),
 	}
 
-	h.Server = stellarsvc.New(g, h, remote.NewRemoteNet(g))
```

Comment:
Previously a new remoter was created for each local rpc server `stellarsvc.Server`.